### PR TITLE
Cleans up

### PR DESCRIPTION
**Pull request by lukaszlenart**
Initial cleanup to allow release a new version

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-[<img src="https://img.shields.io/discord/633198088311537684?color=8899f7&label=DISCORD&style=for-the-badge" height="24">](https://discord.gg/CHCPjERybv)
-[<img src="https://img.shields.io/matrix/propensive.magnolia:matrix.org?label=MATRIX&color=0dbd8b&style=for-the-badge" height="24">](https://app.element.io/#/room/#propensive.magnolia:matrix.org)
-[<img src="https://img.shields.io/twitter/follow/propensive?color=%2300acee&label=TWITTER&style=for-the-badge" height="24">](https://twitter.com/propensive)
-[<img src="https://img.shields.io/maven-central/v/com.propensive/magnolia_2.12?color=2465cd&style=for-the-badge" height="24">](https://search.maven.org/artifact/com.propensive/magnolia_2.12)
-[<img src="https://vent.dev/badge/propensive/magnolia" height="24">](https://vent.dev/)
+[<img alt="GitHub Workflow" src="https://img.shields.io/github/workflow/status/softwaremill/magnolia/Build/main?style=for-the-badge" height="24">](https://github.com/softwaremill/magnolia/actions)
+[<img src="https://img.shields.io/maven-central/v/com.softwaremill/magnolia_2.12?color=2465cd&style=for-the-badge" height="24">](https://search.maven.org/artifact/com.softwaremill/magnolia_2.12)
```
Comment (adamw) [inline readme.md:2]: the group id for the badge should be `com.softwaremill.magnolia` I think

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -1,18 +1,27 @@
 name: Build
 
-on: [push]
+on:
```
Comment (adamw) [inline .github/workflows/main.yml:3]: to be able to release using the sbt-softwaremill plugin you'll also need a second publish-release job - see the `main` branch. It also includes release-drafter

**Review by adamw — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (lukaszlenart) [reply readme.md]: 👍

File: .github/workflows/main.yml (hunk 2 above)
Comment (lukaszlenart) [reply .github/workflows/main.yml:2]: 👍

**Pull request merged**